### ISSUE TITLE
Update README

### ISSUE THREAD

**Issue by justinalexandershaw** [PrepTimer/PrepTime#15]
Update the readme to be pleasing to look at for non-developers. Consider making this repository public and adding automation and checks for PRs.

**Comment by justinalexandershaw** [comment]
PrepTimer - A Promise Kept

*This post was originally posted in the r/debate subreddit and was crossed here to inform a wider audience*


Hey y’all!

A year ago I teamed up with some of you on this subreddit to make a simple debate timer app. That project quickly turned into a project larger and more impactful than I could have ever imagined. One year and 3.6K downloads later, it’s time to fulfill a promise. 

In an attempt to make a truly community-based debate app, last year I promised to donate 100% of all net profits to the Urban Debate League.

It’s been a year. 

So in the spirit of [full disclosure](https://m.imgur.com/a/UR0jayt): here’s the summary of our financials since Apr 2018. 

The app makes money in two ways: 
   1.) Google AdMob (ads on the free version)
   2.) Apple App Store ($0.99 Pro version)

The app only has one yearly expenditure (unless I paid myself, which I haven’t been able to so far) and that expenditure is the Apple Developer Program’s yearly membership ($99.00 plus tax, required to host app on App Store).


TLDR;

Gross Revenue:
+$119.00 - Google AdMob (909K impressions)
+$48.85  - Apple App Store (70 Pro downloads)

Gross Expenditures:
-$108.80 - Apple Developer Yearly Membership 

Net Profit: 
+$59.05 - Donated to Urban Debate League


It’s been a pleasure working with so many of you to implement the ideas you’ve come up with to build what is truly an app built by debaters, for debaters. I have many more ideas that I have been working on and am excited to introduce in the coming year. If you would like to help with that process, you can support my work on [Patreon](https://www.patreon.com/justinshaw), download the Pro version of PrepTime, or click those ads! 

I have really enjoyed working with all of you to develop this tool, and can’t wait to expand it into more platforms and with more tools so that everybody can join in on the fun. 

Thanks again for a great year,

Justin

**Comment by justinalexandershaw** [comment]
PrepTimer Audio

What’s up Debate community!

After just a few, short months of hard work and sleepless nights I am excited to announce that [PrepTimer](https://itunes.apple.com/us/app/prep-timer/id1378824730?mt=8) has grown from an app that I started on the bus ride to State, to an app that has over 750 downloads across four of the seven continents! It is also the number-one rated Speech and Debate app on the App Store! To celebrate, I am proud to announce the release of version 4.1 of [PrepTimer](https://itunes.apple.com/us/app/prep-timer/id1378824730?mt=8)!

The new version includes many new features including: 
* Automatic Audio Recordings of every speech in every round to help you with speech redos to get better. 
* Added confetti animation at the end of every completed speech to boost morale!
* Added a progress bar to the Debate timer to help visualize time allocation. 
* Fixed a couple of small bugs found and reported to me by users in the debate community.

**Comment by justinalexandershaw** [comment]
I'm gonna close this issue for now as of the latest push.
